packages/FormElement/src/FormElement.styles.js

```diff
@@ -8,18 +8,23 @@ const FontSizes = {
 };
 
 export const SectionsContainer = styled.div`
-  ${({ isInline }) => isInline && `flex-grow: 1;`}
+  ${props => `
```

Comment:
I preferred the tidyness of the previous version where this could all be in one line.  I notice that you're consistently avoiding the destructuring of `props`, was there another reason besides consistency?  Was that approcah recommended by someone?